hack/charts/namespaced/operator_chart/values.yaml

```diff
@@ -0,0 +1,5 @@
+roleArn: arn:aws:iam::123456789012:role/DELETE_ME
+
```

Comment:
Can we add an additional option to be able to set which namespace the controller watches, in this values section? For instance if I wanted to install the operator into `my_secret_namespace` but make it watch `default`, there is no way to do that.

Reply:
Oh thats a miss, will do.

Reply:
Why would i want to do that?

Reply:
Some people may install their operators into a namespace that is controlled by the devops or admins. The data scientists would submit their jobs into another namespace that they have control over, or that is tied to them in some way.

Reply:
Yeah we should have an option to pass namespace as well. 
I think behavior for namespace among helm2 and helm3 are different so don't get surprise.

Reply:
spoke in person namespace of operator and job has to be same. Generally helm user create namespace and add resources to it. In this case we might have to have two separate helm chart.
1. cluster scope (we already have this)
2. namespace scoped (this is what we need to do this)

Reply:
Oh? Why does the namespace of the operator and the job have to be the same?

Reply:
@RedbackThomson I think that is a valid but edge case. Even with the current implementation if someone wanted to achieve this, they could modify the templates before deploying the helm. 
On the other hand adding an extra namespace option to the values.yaml file would be confusing to the majority of the users. I say this after reading posts on helm usage ([link this one](https://github.com/helm/helm/issues/5628)) which indicate that helm2 vs helm3 already adds some confusion about which namespace helm deploys resources to. Lets keep it simple for the majority.

Reply:
Sure! Totally reasonable, yep.

Reply:
> Oh? Why does the namespace of the operator and the job have to be the same?

don't we have operator running in namespace will pick the job only in that namespace ?

Reply:
> > Oh? Why does the namespace of the operator and the job have to be the same?
> 
> don't we have operator running in namespace will pick the job only in that namespace ?

The namespace in which the operator is deployed does not need to be the one it monitors. They are separate fields that can be set independently. Whether RBAC permits that is a different question, but in theory it's entirely possible.

Reply:
Yeah thats definitely confusing, let user define the namespace if they want it separate or an option.